group-coordinator/src/main/java/org/apache/kafka/coordinator/group/Group.java

```diff
@@ -90,4 +90,29 @@ void validateOffsetFetch(
         int memberEpoch,
         long lastCommittedOffset
     ) throws KafkaException;
+
+    /**
+     * Validates the OffsetDelete request.
+     */
+    void validateOffsetDelete() throws KafkaException;
+
+    /**
+     * Validates the GroupDelete request
+     */
+    void validateGroupDelete() throws KafkaException;
+
+    /**
+     * Returns true if the group is actively subscribed to the topic.
+     *
+     * @param topic the topic name.
+     * @return whether the group is subscribed to the topic.
+     */
+    boolean isSubscribedToTopic(String topic);
```

Comment:
Also we decided to use topicIds throughout the protocol, are we using topic names for this API?

Reply:
Offsets APIs still use topic names...

Reply:
got it okie!